### PR TITLE
chore: quay ci

### PR DESCRIPTION
**Pull request by nkoppisetty**
Adding CI with quay.io, for publishing and releasing images

### CODE REVIEW

**Review by ab93 — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,36 +1,56 @@
...
-        cache: 'poetry'
+        registry: quay.io
+        username: ${{ secrets.QUAYIO_USERNAME }}
+        password: ${{ secrets.QUAYIO_PASSWORD }}
 
-    - name: Install dependencies
-      run: |
-        poetry env use ${{ matrix.python-version }}
-        poetry install --with dev --all-extras
+    - name: Docker Buildx Argo workflow
```
Comment (whynowy) [inline .github/workflows/ci.yml:26]: xxxx

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,36 +1,56 @@
...
+      env:
+        DOCKERIO_ORG: quay.io/numaio
+        LATFORM: linux/x86_64
+        TARGET: numalogic-prometheus/udf
+        run: |
+          tag=$(basename $GITHUB_REF)
+          if [ $tag = "master" ]; then
```
Comment (whynowy) [inline .github/workflows/ci.yml:33]: `main`

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1,36 +1,56 @@
...
+            tag="latest"
+          fi
+          
+          image_name="${DOCKERIO_ORG}/numalogic-prometheus/udf:${tag}"
```
Comment (whynowy) [inline .github/workflows/ci.yml:37]: Use `QUAYIO_ORG` maybe.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (whynowy) [inline .github/workflows/ci.yml:26]: xxxxx

File: .github/workflows/ci.yml (hunk 2 above)
Comment (whynowy) [inline .github/workflows/ci.yml:33]: `main`

**Review by whynowy — APPROVED** [review]

**Pull request merged**